.renovaterc.json

```diff
@@ -17,82 +17,95 @@
   "separateMinorPatch": true,
   "packageRules": [
     {
-      "updateTypes": ["minor"],
-      "groupName": "subpackage dependencies"
+      "paths": ["**/*"],
+      "groupName": "subpackage devDependencies",
```

Comment:
Updates are grouped based on their branch ex `renovate-bot:renovate/patch-subpackage-dependencies`. This group name is used to determine the branch name. `separateMinorPatch` causes the `patch` prefix to be included.

Major updates with no `groupName` configured are named by the dependency and version, ex `renovate-bot:renovate/google-closure-compiler-20200517.x`. That's why multiple updates of the same dependency get grouped: they live on the same branch.

This adds a branch prefix for each of the package groups, so we'd get both `renovate-bot:renovate/core-google-closure-compiler-20200517.x` and `renovate-bot:renovate/validator-google-closure-compiler-20200517.x`

Reply:
Gotcha.

Style nit: Would it be clearer if the `managerBranchPrefix`es ended in a forward slash (`renovate-bot:renovate/core/google-closure-compiler-20200517.x`)?

Right now, the dash makes it look like the package name is `validator-google-closure-compiler-20200517.x` and not `google-closure-compiler-20200517.x`

Reply:
While I have no good reason to believe including a slash would break anything, my gut is a tad reluctant to try it right now. I'd prefer to validate that the config and grouping itself works as desired, and will happily experiment with tweaks like that in a follow-up.I'd rather stick to the functional bits for now, especially since this branch name is rarely seen or read by anyone.

Reply:
(an underscore is also a plausible choice)

Reply:
Just saw the results of this PR (#28521 and #28522). I now see that it only affects the PR branch name and nothing else, so agree that it can be left as is.